### PR TITLE
DLK-12572 Updated docker-wkhtmltopdf-aas 

### PR DESCRIPTION
**Pull request by mmarkovic-mio**
Solution for buffer overflow issue causing child wkhtmltopdf process to fail.

Upgraded libraries to use latest wkhtmltopdf version using a commit from another fork as reference: 
https://github.com/leknoppix/docker-wkhtmltopdf-aas/commit/74dcc79bfa7009ac3e8ba5afcd4138cf62de5285

Added environment variable for enabling verbose and info level logging.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,60 +1,45 @@
-FROM ubuntu:18.04
-MAINTAINER Fabian Beuke <beuke@traum-ferienwohnungen.de>
+FROM ubuntu:20.04 
 
 RUN apt-get update &&                          \
     apt-get install -y --no-install-recommends \
+    add-apt-key                                \
     fontconfig                                 \
-    libcurl3                                   \
+    curl				                       \
```
Comment (cgrohn-mio) [inline Dockerfile]: nit: out of line \

File: Dockerfile [hunk 2]
```diff
@@ -1,60 +1,45 @@
...
+    -o /tmp/nodesource_setup.sh && bash /tmp/nodesource_setup.sh    && \
+    rm /tmp/nodesource_setup.sh                                     && \
+    apt-get install -y --no-install-recommends nodejs               && \
+    wget -q $WK_URL/$WK_PKG && dpkg -i $WK_PKG && rm $WK_PKG        && \
+    rm /usr/local/bin/wkhtmltoimage                                 && \
+    npm install -g coffeescript forever bootprint bootprint-openapi && \
+    bootprint openapi swagger.yaml documentation                    && \
+    npm uninstall -g bootprint bootprint-openapi
+
+RUN npm update       && \
```
Comment (cgrohn-mio) [inline Dockerfile:38]: should we run npm update before the previous run command?

File: Dockerfile (hunk 1 above)
Comment (mmarkovic-mio) [reply Dockerfile]: Took it verbatim from the commit on the other fork. Serves me right.

File: Dockerfile (hunk 2 above)
Comment (mmarkovic-mio) [reply Dockerfile:38]: The above run command runs `npm install`. Running `npm update` here should be fine.

**Review by btorode — APPROVED** [review]

**Pull request merged**